### PR TITLE
fix(orbos): incorporate orbiter into project orbos as base

### PR DESCRIPTION
**Pull request by stebenz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,152 +1,4 @@
-# Orbiter The Meta Cluster Manager
+# Orbos the GitOps Cloud-Automation Project
 
-[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
-[![Release](https://github.com/caos/orbiter/workflows/Release/badge.svg)](https://github.com/caos/orbiter/actions)
-[![license](https://badgen.net/github/license/caos/orbiter/)](https://github.com/caos/orbiter/blob/master/LICENSE)
-[![release](https://badgen.net/github/release/caos/orbiter/stable)](https://github.com/caos/orbiter/releases)
-[![Go Report Card](https://goreportcard.com/badge/github.com/caos/orbiter)](https://goreportcard.com/report/github.com/caos/orbiter)
```
Comment (fforootd) [inline README.md]: Why do we not reuse those?

Comment (stebenz) [reply README.md]: they are still there, just currently moved to https://github.com/caos/orbos/blob/orbos/docs/orbiter.md
will be combined with the banners of boom for the orbos README later

Comment (eliobischof) [reply README.md]: Reuse License too please

File: docs/orbiter.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+The full functionality of the operator is and stays open source and free to use for everyone. We pay our wages by using Orbiter for selling further workload enterprise services like support, monitoring and forecasting, IAM, CI/CD, secrets management etc. Visit our [website](https://caos.ch) and get in touch.
+
+See the exact licensing terms [here](../LICENSE)
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+<<<<<<< HEAD
+
+=======
+>>>>>>> c1d026df6a7d75a2c8f6fbd8733cec699ab8eeb4
```
Comment (eliobischof) [inline docs/orbiter.md]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (fforootd) [reply README.md]: I would delete them, and place them `including` the license on the root. Otherwise I find it confusing

**Review by eliobischof — APPROVED** [review]
Follow-up issue: #93

File: README.md [hunk 3]
```diff
@@ -1,152 +1,4 @@
...
-
-## License
-
-The full functionality of the operator is and stays open source and free to use for everyone. We pay our wages by using Orbiter for selling further workload enterprise services like support, monitoring and forecasting, IAM, CI/CD, secrets management etc. Visit our [website](https://caos.ch) and get in touch.
-
-See the exact licensing terms [here](./LICENSE)
-
-Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+The project contains currently following operators:
+ - [Orbiter](docs/orbiter.md)
```
Comment (eliobischof) [inline README.md:4]:
```sugestion
## License

The full functionality of the operator is and stays open source and free to use for everyone. We pay our wages by using Orbiter for selling further workload enterprise services like support, monitoring and forecasting, IAM, CI/CD, secrets management etc. Visit our [website](https://caos.ch) and get in touch.	

See the exact licensing terms [here](./LICENSE)	

Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
```

File: README.md [hunk 4]
```diff
@@ -1,152 +1,4 @@
-# Orbiter The Meta Cluster Manager
+# Orbos the GitOps Cloud-Automation Project
```
Comment (eliobischof) [inline README.md:2]:
```suggestion

[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
[![Release](https://github.com/caos/orbiter/workflows/Release/badge.svg)](https://github.com/caos/orbiter/actions)
[![license](https://badgen.net/github/license/caos/orbiter/)](https://github.com/caos/orbiter/blob/master/LICENSE)
[![release](https://badgen.net/github/release/caos/orbiter/stable)](https://github.com/caos/orbiter/releases)
[![Go Report Card](https://goreportcard.com/badge/github.com/caos/orbiter)](https://goreportcard.com/report/github.com/caos/orbiter)
```

File: README.md [hunk 5]
```diff
@@ -1,152 +1,4 @@
...
```
Comment (eliobischof) [inline README.md]:
```suggestion
# Orbos - GitOps everything

[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
[![Release](https://github.com/caos/orbiter/workflows/Release/badge.svg)](https://github.com/caos/orbiter/actions)
[![license](https://badgen.net/github/license/caos/orbiter/)](https://github.com/caos/orbiter/blob/master/LICENSE)
[![release](https://badgen.net/github/release/caos/orbiter/stable)](https://github.com/caos/orbiter/releases)
[![Go Report Card](https://goreportcard.com/badge/github.com/caos/orbiter)](https://goreportcard.com/report/github.com/caos/orbiter)

> This project is in alpha state. The API will continue breaking until version 1.0.0 is released

## [Orbiter](docs/orbiter.md)

## License

The full functionality of the operator is and stays open source and free to use for everyone. We pay our wages by using Orbiter for selling further workload enterprise services like support, monitoring and forecasting, IAM, CI/CD, secrets management etc. Visit our [website](https://caos.ch) and get in touch.	

See the exact licensing terms [here](./LICENSE)	

Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
```

File: README.md (hunk 3 above)
Comment (eliobischof) [inline README.md]:
```suggestion
```

File: docs/orbiter.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
+# Orbiter The Meta Cluster Manager
+
+[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
+[![Release](https://github.com/caos/orbos/workflows/Release/badge.svg)](https://github.com/caos/orbos/actions)
+[![license](https://badgen.net/github/license/caos/orbos/)](https://github.com/caos/orbos/blob/master/LICENSE)
+[![release](https://badgen.net/github/release/caos/orbos/stable)](https://github.com/caos/orbos/releases)
+[![Go Report Card](https://goreportcard.com/badge/github.com/caos/orbos)](https://goreportcard.com/report/github.com/caos/orbos)
+
+> This project is in alpha state. The API will continue breaking until version 1.0.0 is released
+
```
Comment (eliobischof) [inline docs/orbiter.md]:
```suggestion
```

File: docs/orbiter.md [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+See [contribute](orbiter/contribute.md) for details
+
+## License
+
+The full functionality of the operator is and stays open source and free to use for everyone. We pay our wages by using Orbiter for selling further workload enterprise services like support, monitoring and forecasting, IAM, CI/CD, secrets management etc. Visit our [website](https://caos.ch) and get in touch.
+
+See the exact licensing terms [here](../LICENSE)
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
```
Comment (eliobischof) [inline docs/orbiter.md]:
```suggestion
```

**Review by eliobischof — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (eliobischof) [reply README.md]: done

**Review by eliobischof — APPROVED** [review]

**Pull request merged**